src/connectors/README.md

```diff
@@ -114,6 +120,9 @@ const MyComponent = ({ isVisible, isLoading, pattern }) => {
   if (!isVisible) {
     return null;
   }
+  if (state.title !== 'example title') {
```

Comment:
This example is pretty weird, but I wouldn't reject this PR because of this. Just wanted to point this out ;) as a suggestion.

Reply:
definitely a weird example. Thanks for the suggestion.